packages/testing/constants.ts

```diff
@@ -40,6 +40,10 @@ export const testConfigs = {
     id: 'ozylzc5m',
     label: 'Cleeng SVOD',
   },
+  profiles: {
+    id: '4dacztho',
+    label: 'Inplayer Authvod + Profiles',
+  },
```

Comment:
Is this something we want to merge? Or is this a accident?

Reply:
I found it useful actually. But I doubt now, because I think profiles have been switched off again (and we might want to create a demo config with profiles actually).